requirements-test.txt

```diff
@@ -1,6 +1,6 @@
 # This only installs PyTorch with a specific CUDA version which may not be
 # compatible with yours. If so, install PyTorch with the correct CUDA version
 # as instructed on https://pytorch.org/get-started/locally/
-torch
+torch>=1.10.0
```

Comment:
The PyTorch version that gets installed for testing is specified in `.github/workflows/run_tests.yml`. You should change the version there rather than here. That said, I'm not comfortable with testing only for v1.10.0 just to accommodate torch scripting. I think the ideal option is to run all tests *except* `TestTorchScript` for v1.2, and run *only* `TestTorchScript` on v1.10.0. That way, all the other tests are still run for <v1.10.0, ensuring that the library still supports those earlier versions. Would you be interested in implementing this?